pages/common/xh.md

```diff
@@ -0,0 +1,28 @@
+# xh
+
+> XH: Friendly and fast tool for sending HTTP requests.
+> More information: <https://github.com/ducaale/xh>.
+
+- Send a GET request:
+
+`xh {{httpbin.org/get}}`
+
+- Send a POST request with JSON body: `{"name": "ahmed", "age": 25}`:
+
+`xh post {{httpbin.org/post}} {{name=john}} {{age:=25}}`
```

Comment:
Missing colon?
```suggestion
`xh post {{httpbin.org/post}} {{name:=john}} {{age:=25}}`
```

Reply:
> Missing colon?

No, thats expected, numbers use `:=` syntax

Reply:
Ah, good to know. Thank you.